src/tools/illink/test/Mono.Linker.Tests/TestCasesRunner/TrimmingDriver.cs

```diff
@@ -5,13 +5,13 @@
 
 namespace Mono.Linker.Tests.TestCasesRunner
 {
-	public class LinkerDriver
+	public class TrimmingDriver
```

Comment:
Even more reason to call the shared stuff `Trimming.Shared`/`Trimming.Tests.Shared` ;-).